engines/dgds/ttm.cpp

```diff
@@ -934,8 +917,8 @@ void TTMInterpreter::handleOperation(TTMEnviro &env, TTMSeq &seq, uint16 op, byt
 		doWipeOp(op, env, seq, Common::Rect(Common::Point(ivals[0], ivals[1]), ivals[2], ivals[3]));
 		break;
 	case 0xa0a0: { // DRAW LINE  x1,y1,x2,y2:int
-		ClipSurface clipSurf(seq._drawWin, _vm->_compositionBuffer.surfacePtr());
-		Graphics::drawLine(ivals[0], ivals[1], ivals[2], ivals[3], seq._drawColFG, plotClippedPoint, &clipSurf);
+		Graphics::Surface clipSurf(_vm->_compositionBuffer.getSubArea(seq._drawWin));
```

Comment:
Wouldn't this require translating the parameters to `drawLine` and `drawPolygonScan`?

Reply:
Yes, you are right. Thank you!
Fixed.